ui/apps/platform/src/Containers/Policies/Wizard/Step6/AttachNotifiersForm.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback } from 'react';
+import { Table, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { Button, Divider, Flex, Form, Title } from '@patternfly/react-core';
+import { useField } from 'formik';
+
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+import useRestQuery from 'hooks/useRestQuery';
+import { integrationsPath } from 'routePaths';
+import { fetchNotifierIntegrations } from 'services/NotifierIntegrationsService';
+import { getTableUIState } from 'utils/getTableUIState';
+
+function AttachNotifiersForm() {
+    const [field, , helpers] = useField('notifiers');
+
+    const fetchNotifiers = useCallback(() => fetchNotifierIntegrations(), []);
+    const { data: notifiers = [], isLoading, error } = useRestQuery(fetchNotifiers);
+
+    const tableState = getTableUIState({
+        isLoading,
+        data: notifiers,
+        error,
+        searchFilter: {},
+    });
+
+    function onSelectNotifier(e, isSelected, rowIndex) {
+        const selectedNotifiers = field.value || [];
+        if (isSelected) {
+            helpers.setValue([...selectedNotifiers, notifiers[rowIndex].id]);
+        } else {
+            helpers.setValue(selectedNotifiers.filter((id) => id !== notifiers[rowIndex].id));
+        }
+    }
+
+    function onSelectAllNotifiers(e, isSelected) {
+        if (isSelected) {
+            helpers.setValue(notifiers.map((notifier) => notifier.id));
+        } else {
+            helpers.setValue([]);
+        }
+    }
+
+    return (
+        <Flex
+            direction={{ default: 'column' }}
+            spaceItems={{ default: 'spaceItemsNone' }}
+            flexWrap={{ default: 'nowrap' }}
+        >
+            <Flex
+                direction={{ default: 'column' }}
+                spaceItems={{ default: 'spaceItemsSm' }}
+                className="pf-v5-u-p-lg"
+            >
+                <Title headingLevel="h2">Notifiers</Title>
+                <div>
+                    Forward policy violations to external tooling by selecting one or more notifiers
+                    from existing integrations.
+                </div>
+            </Flex>
+            <Divider component="div" />
+            <Form>
+                <div className="pf-v5-u-p-lg">
+                    <Table aria-label="Attach notifiers table" borders>
```

Comment:
This `aria-label` prop here is questionable when there is a visible `Title` element.